### DISCUSSION TITLE
Twig filter for cache busting of css and js files

### DISCUSSION THREAD

**Discussion by michel-o** [craftcms/cms#8572] [Ideas]
This twig filter would take in path to specific file, and output it with GET parameter, consisting of file modification time (timestamp). 

Each time file would change, GET parameter would also change. Ideal for browser cache busting.

For example this:
`<script src="{{'scripts/script.js'|updateFile}}"></script>`
Outputs:
`<script src="scripts/script.js?v=1234556789990"></script>`

**Comment by brandonkelly** [comment]
The filter would also need to know the file’s server path in order to know its modify time.

**Comment by michaelrog** [comment]
André Elvan's [Stamp](https://github.com/aelvan/Stamp-Craft/tree/craft3) plugin works marvelously for accomplish exactly this.

**Comment by XhmikosR** [comment]
I second this FR, but instead of using the modified time, maybe using a hash?
